certmanager/checkupdate.py

```
@@ -0,0 +1,271 @@
+    auth_data = {
+        "id": "string",
+        "metadata": {},
+        "request": {
+            "appId": "ida",
+            "clientId": "mosip-deployment-client",
+            "secretKey": client_secret
+        },
+        "requesttime": get_utc_timestamp(),
+        "version": "string"
+    }
+    response = requests.post(auth_url, headers=headers, json=auth_data)
```

Ruff (0.16.3)

[error] Probable use of `requests` call without timeout

(S113)